screens/clubAnnouncementPage.js

```diff
@@ -54,55 +57,66 @@ class clubAnnouncementPage extends Component{
+    const events = this.state.events
     return(
-      <SafeAreaView style={{height: '100%'}}>
-        <ScrollView>
-          <View>
-            <View style={styles.titleContainer}>
-              <Title style={styles.title}>
-                  Announcements
-              </Title>
-            </View>
-            <View style={styles.container}>
-              <Card
-                style={styles.card}>
-                <Card.Title
-                  subtitle={announcement}
-                  subtitleStyle={styles.clubAnnouncement}
-                  subtitleNumberOfLines={2}
-                />
-              </Card>
-            </View>
-            <View style={styles.titleContainer}>
-              <Title style={styles.title}>
-                Events
-              </Title>
-            </View>
-            <View style={styles.container}>
-              <Card
-                style={styles.card}>
-                <Card.Title
-                  subtitle={announcement}
-                  subtitleStyle={styles.clubAnnouncement}
-                  subtitleNumberOfLines={2}
-                />
-              </Card>
-              </View>
-            <View style={styles.row}>
-            <Button mode="contained" dark="true" style={styles.button} onPress = {this.createAnnouncement} > Create Announcement </Button>
-            </View>
-            <View style={styles.row}>
-            <Button mode="contained" dark="true" style={styles.button} onPress = {this.createEvent} > Create Event </Button>
-            </View>
-            <View style={styles.row}>
-            <Button mode="contained" dark="true" style={styles.button} onPress = {this.goToChat} > Go To Group Chat </Button>
-            </View>
-          </View>
-        </ScrollView>
-      </SafeAreaView>
+        <View style={{height: "100%", width: "100%", backgroundColor: "lightgray"}}>
+                <View style={{alignItems: "center"}}>
+                    <View style={styles.titleContainer}>
+                        <Title style={styles.title}>
+                            Announcements
+                        </Title>
+                    </View>
+                    <View style={styles.container}>
+                        <Card
+                            style={styles.card}>
+                            <Card.Title
+                                subtitle={announcement}
+                                subtitleStyle={styles.clubAnnouncement}
+                                subtitleNumberOfLines={2}
+                            />
+                        </Card>
+                    </View>
+                    <View style={styles.titleContainer}>
+                        <Title style={styles.title}>
+                            Events
+                        </Title>
+                    </View>
+                    <View style={{ height: "30%", width: "95%", backgroundColor: "#4169E1", borderRadius: 6, margin: 5 }}>
```

Comment:
We can only see 1.5 card at the moment. Would it be possible to make the view larger so that user can see 3 cards?